docker/Dockerfile

```diff
@@ -1,34 +1,127 @@
-ARG BASE_IMAGE=python:3.8-slim-bullseye
+##
+#  Generic dockerfile for dbt image building.  
+#  See README for operational details
+##
 
-FROM $BASE_IMAGE
-ARG BASE_REQUIREMENTS_SRC_PATH
-ARG WHEEL_REQUIREMENTS_SRC_PATH
-ARG DIST_PATH
+##
+# base image (abstract)
+##
+FROM python:3.9-slim-bullseye as base
```

Comment:
not a requirement since we never used it for our purposes, but is there a reason we removed the ability to override the base image with a build arg?

Reply:
Just because I didn't see a use case where we needed it.  :)
I'm happy to put it back if you have feels about it.

Reply:
all good!